### DISCUSSION TITLE
How to pull the latest image for using a tag in docker-compose?

### DISCUSSION THREAD

**Discussion by patrickdung** [containers/podman#12288] [Q&A] [answered]
So the command is podman-compose.
I had defined this in docker-compose.yaml
```
   image: docker.io/postgres:13-bullseye
```
The image was PGSQL 13.4. Now 13.5 is released and the image is update in Docker Hub.
I tried to run "podman-compose up -d --pull-always"
But the latest image is not pulled.

Any ideas about the problem? Thanks in advance.

**Comment by mheon** [comment, marked as answer]
Please ask this in the podman-compose repository - https://github.com/containers/podman-compose

This is likely an issue with podman-compose and not Podman.

**Reply by patrickdung** [reply]
Ok. I had open a [FR](https://github.com/containers/podman-compose/issues/356) in podman-compose.
